### ISSUE TITLE
deadband

### ISSUE THREAD

**Issue by demirciberkan** [PowerBroker2/ArduPID#17]
Can you explain what would be the deadband please.

As i understand it stands for the gap which is close to the bias point and doesnt actually move the motor.
As an example if we set the output limits to -255 , 255 then if the motor can not actually move in between -30 , 30 deadband should recalculate these values and  only use  the values between -30 -255 and 30 255 is that right ?

And i can not see its usage in the code. Not implemented yet ?
Thanks in advance

**Comment by PowerBroker2** [comment]
You're right, it seems I added all the architecture for deadband and then forgot to implement or something. Been a long while since I worked on this codebase, so I forget how this particular piece was supposed to work. However, I think it's supposed to work as you describe...
